### PR TITLE
Remove triple backticks

### PR DESCRIPTION
**Pull request by jayvdb**
Includes https://github.com/coala/coala-bears/pull/1555

Fixes https://github.com/coala/coala-bears/issues/1561 and https://github.com/coala/coala-bears/issues/1544

### CODE REVIEW

**Comment by yash-nisar** [comment]
Awesome ! 🎉

**Comment by yash-nisar** [comment]
ack 8ccc1ea81ad1e1e7c46e85521c4684aea46b7a02

**Comment by jayvdb** [comment]
@rultor merge

File: bears/coffee_script/CoffeeLintBear.py [hunk 1]
```diff
@@ -133,69 +133,68 @@ def generate_config(filename, file,
...
-            ```
-            are prohibited, whereas
-            ```
-            @alright
-            @(fn)
-            @ok()
-            @[ok]
-            @ok()
-            ```
-            are accepted.
```
Comment (supergr35) [inline bears/coffee_script/CoffeeLintBear.py]: Why the omission of `are accepted` ?

**Comment by jayvdb** [comment]
@rultor stop

File: bears/coffee_script/CoffeeLintBear.py (hunk 1 above)
Comment (jayvdb) [inline bears/coffee_script/CoffeeLintBear.py]: Thank you for carefully reviewing.  Could you do a follow up patch, re-adding it?  Be sure to mention this commit in your commit message, and use the same issue #.

Comment (supergr35) [inline bears/coffee_script/CoffeeLintBear.py]: Sorry, I missed the ack. 
Should I file an issue (would be good for newcomers) to add this line back or leave it be?
CC: @jayvdb

Comment (jayvdb) [inline bears/coffee_script/CoffeeLintBear.py]: Being fixed in https://github.com/coala/coala-bears/pull/1563

**Pull request merged**